### PR TITLE
remove duplicate events.

### LINKED ISSUE #3365: events listed twice in subprojects
the events subproject on the Contributor Experience README is listed under community management and its own category of events. good first issue for someone to remove from under the community management and leave as a stand alone subproject. 

[README](https://github.com/kubernetes/community/tree/master/sig-contributor-experience)

### PR DESCRIPTION
**Pull request by tao12345666333**
fix #3365

Signed-off-by: Jintao Zhang <zhangjintao9020@gmail.com>

### CODE REVIEW

**Comment by SeanPSnyder** [comment]
Oktotest

File: sig-contributor-experience/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The following subprojects are owned by sig-contributor-experience:
     - https://raw.githubusercontent.com/kubernetes/community/master/OWNERS
 - **community-management**
   - Owners:
-    - https://raw.githubusercontent.com/kubernetes/community/master/communication/OWNERS https://raw.githubusercontent.com/kubernetes/community/master/events/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/community/master/communication/OWNERS
```
Comment (nikhita) [inline sig-contributor-experience/README.md]: This is generated code using `sigs.yaml`. :)

You'll need to remove the line here:

https://github.com/kubernetes/community/blob/3bfd133d8624525326c62e809eb59c4eb0bcfb9b/sigs.yaml#L988

and then run `make generate`.

Comment (tao12345666333) [reply sig-contributor-experience/README.md]: Thanks. Fixed.

**Comment by nikhita** [comment]
/assign @parispittman 
for approval

**Comment by tao12345666333** [comment]
PTAL @parispittman Thanks. :heart:

**Comment by parispittman** [comment]
thank you @tao12345666333! appreciate you doing this

/lgtm
/approve

**Pull request merged**